src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -678,12 +682,63 @@ void GraphOptimizer::FuseMultiplyAndAdd(Graph& graph) {
     }
 }
 
-void GraphOptimizer::MergeConvertAndScaleShift(Graph& graph) {
+void GraphOptimizer::MergeEltwiseAndConvert(Graph& graph) {
+// The pass is required on arm platforms only
+#if !defined(OPENVINO_ARCH_ARM64)
+    return;
+#endif
+    auto& graphNodes = graph.GetNodes();
+
+    auto parent = graphNodes.begin();
+    while (parent != graphNodes.end()) {
+        CPU_GRAPH_OPTIMIZER_SCOPE(MergeEltwiseAndConvert);
+        auto parentNode = *parent;
+        if (parentNode->getType() != Type::Eltwise) {
+            parent++;
+            continue;
+        }
+
+        const auto& childEdges = parentNode->getChildEdges();
+        if (childEdges.size() != 1) {
+            parent++;
+            continue;
+        }
+
+        const auto edge = childEdges[0].lock();
+        auto childNode = edge->getChild();
+        if (childNode->getType() != Type::Convert) {
+            parent++;
+            continue;
+        }
+
+        if (!one_of(childNode->getOriginalOutputPrecisionAtPort(0),
+                    ov::element::i8,
+                    ov::element::u8,
+                    ov::element::f16,
+                    ov::element::bf16,
+                    ov::element::f32)) {
+            parent++;
+            continue;
+        }
+
+        auto fusedOps = parentNode->getFusedWith();
```

Comment:
Lets mention here that this is a WA, because Eltwise node uses precision of last fused node as output precision.

Reply:
Added the comment